pkg/agent/route/route_windows.go

```diff
@@ -36,29 +36,37 @@ const (
 )
 
 type Client struct {
-	nr          netroute.Interface
-	nodeConfig  *config.NodeConfig
-	serviceCIDR *net.IPNet
-	hostRoutes  *sync.Map
-	fwClient    *winfirewall.Client
+	nr             netroute.Interface
+	nodeConfig     *config.NodeConfig
+	networkConfig  *config.NetworkConfig
+	serviceCIDR    *net.IPNet
+	hostRoutes     *sync.Map
+	fwClient       *winfirewall.Client
+	bridgeInfIndex int
 }
 
 // NewClient returns a route client.
 // Todo: remove param serviceCIDR after kube-proxy is replaced by Antrea Proxy completely.
 func NewClient(serviceCIDR *net.IPNet, networkConfig *config.NetworkConfig, noSNAT bool) (*Client, error) {
 	nr := netroute.New()
 	return &Client{
-		nr:          nr,
-		serviceCIDR: serviceCIDR,
-		hostRoutes:  &sync.Map{},
-		fwClient:    winfirewall.NewClient(),
+		nr:            nr,
+		networkConfig: networkConfig,
+		serviceCIDR:   serviceCIDR,
+		hostRoutes:    &sync.Map{},
+		fwClient:      winfirewall.NewClient(),
 	}, nil
 }
 
 // Initialize sets nodeConfig on Window.
 // Service LoadBalancing is provided by OpenFlow.
 func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 	c.nodeConfig = nodeConfig
+	if bridgeInf, err := net.InterfaceByName(nodeConfig.OVSBridge); err != nil {
+		klog.Warningf("Failed to find the interface %s: %v", nodeConfig.OVSBridge, err)
```

Comment:
Do we need to fail if noEncap/hybrid is configured?

Reply:
Yes, changed klog warning to an error.